src/deepsparse/yolo/pipelines.py

```diff
@@ -285,6 +286,22 @@ def _make_channels_first(self, image: numpy.ndarray) -> numpy.ndarray:
 
         return image
 
+    def _make_channels_last(self, image: numpy.ndarray) -> numpy.ndarray:
```

Comment:
I have the exact same function in my yolact utils. It hurts to see how we duplicate the CV code. Just to reinforce your ask from yesterday @rahul-tuli (about refactoring the vision pipelines).

Reply:
@dbogunowicz I agree but, I did not want to import `yolact` utils inside `yolo` pipelines, in my opinion these helper functions need to be refactored out into a parent package and they need to be tested for all three cv pipelines which is non-trivial work;

My ask yesterday was to refactor CV Input Schemas which is comparatively very trivial, all CV input schemas had the exact same function with same lines of code, that was moved to a parent class.

I propose to leave this as is here, and take up refactoring as a separate bigger task. Thoughts?